tensorrt_llm/_torch/models/modeling_deepseekv3.py

```diff
@@ -167,10 +170,19 @@ def forward(self,
             else:
                 hidden_states = hidden_states[-1].unsqueeze(0)
 
-        if not (self.model_config.mapping.enable_attention_dp):
```

Comment:
Thanks for the change.
Lets hold this PR , not merge to this SA branch, and only target to main branch. 
Lets stablize the branch. Thx

Reply:
Yes,target to main as we offline discussed.